pkg/kubectl/delete.go

```diff
@@ -226,6 +226,17 @@ func (reaper *ReplicaSetReaper) Stop(namespace, name string, timeout time.Durati
 		timeout = Timeout + time.Duration(10*rs.Spec.Replicas)*time.Second
 	}
 
+	_, err = reaper.updateReplicaSetWithRetries(namespace, name, func(rs *extensions.ReplicaSet) {
+		// set replicaset's initializers to nil
```

Comment:
chasing disabling initializers on everything seems very error-prone. @smarterclayton, what would you expect to happen here?

Reply:
I definitely think that this is a bug in the reaper.  Why do we still have the reaper?

A normal user isn't going to be able to perform this operation, so this isn't the correct fix.  I think if the object hasn't been initialized yet the reaper should just call delete.